### DISCUSSION TITLE
MayaUSD roadmap

### DISCUSSION THREAD

**Discussion by hamedsabri** [Autodesk/maya-usd#2614] [General]
Hi guys, 

I hope everyone is doing well. We currently have a few people at Weta Digital who are interested in learning more about MayaUSD roadmap and would like to participate in MayaUSD meetings that we used to run with different studios. 

I would like to know if these meetings still run? and what is the process to join?

Cheers,
-Hamed

**Comment by tanant** [comment]
Heya Hamed - we're in the process of trying to get some of these set back up, so lemme get back to you on this one. Regardless, I'm your best contact on this so feel free to bug me.
